hello_nexus/basic/handler/service_handler.py

```diff
@@ -0,0 +1,86 @@
+alternative "fully manual" style where you implement an OperationHandler class directly.
+"""
+
+from __future__ import annotations
+
+import uuid
+
+from nexusrpc.handler import (
+    OperationHandler,
+    StartOperationContext,
+    SyncOperationHandler,
+    operation_handler,
+    service_handler,
+)
+from temporalio.nexus.handler import (
+    WorkflowOperationToken,
+    WorkflowRunOperationHandler,
+    start_workflow,
+)
+
+from hello_nexus.basic.handler.db_client import MyDBClient
+from hello_nexus.basic.handler.workflows import WorkflowStartedByNexusOperation
+from hello_nexus.basic.service import MyInput, MyNexusService, MyOutput
+
+
+@service_handler(service=MyNexusService)
+class MyNexusServiceHandler:
+    # You can create an __init__ method accepting what is needed by your operation
+    # handlers to handle requests. You typically instantiate your service handler class
+    # when starting your worker. See hello_nexus/basic/handler/worker.py.
+    def __init__(self, connected_db_client: MyDBClient):
+        # `connected_db_client` is intended as an example of something that might be
+        # required by your operation handlers when handling requests, but is only
+        # available at worker-start time.
+        self.connected_db_client = connected_db_client
+
+    # This is a nexus operation that is backed by a Temporal workflow. The start method
+    # starts a workflow, and returns a nexus operation token. Meanwhile, the workflow
+    # executes in the background; Temporal server takes care of delivering the eventual
+    # workflow result (success or failure) to the calling workflow.
+    #
+    # The token will be used by the caller if it subsequently wants to cancel the Nexus
+    # operation.
+    @operation_handler
+    def my_workflow_run_operation(
+        self,
+    ) -> OperationHandler[MyInput, MyOutput]:
+        async def start(
+            ctx: StartOperationContext, input: MyInput
+        ) -> WorkflowOperationToken[MyOutput]:
+            # You could use self.connected_db_client here.
+            return await start_workflow(
+                WorkflowStartedByNexusOperation.run,
+                input,
+                id=str(uuid.uuid4()),
+            )
+
+        return WorkflowRunOperationHandler.from_callable(start)
+
+    # This is a Nexus operation that responds synchronously to all requests. That means
+    # that unlike the workflow run operation above, in this case the `start` method
+    # returns the final operation result.
+    #
+    # Here it is implemented using SyncOperationHandler.from_callable.
+    # See service_handler_with_operation_handler_classes.py for an alternative style
+    # involving subclassing SyncOperationHandler and overriding the start method.
+    #
+    # Sync operations are free to make arbitrary network calls, or perform CPU-bound
+    # computations. Total execution duration must not exceed 10s.
+    @operation_handler
```

Comment:
I recall a `@sync_operation_handler` of sorts that would have saved lines here. Did we get rid of that? Do we expect users writing very simple synchronous Nexus ops to have to make nested functions or classes?